chain/network/src/store/mod.rs

```diff
@@ -146,3 +141,27 @@ impl Store {
         self.0.iter::<schema::Peers>().collect::<Result<_, _>>().map_err(Error)
     }
 }
+
+impl std::convert::From<near_store::Store> for Store {
+    fn from(store: near_store::Store) -> Self {
+        Self(schema::Store::new(store))
+    }
+}
+
+impl std::convert::From<&near_store::Store> for Store {
+    fn from(store: &near_store::Store) -> Self {
+        Self(schema::Store::new(store.clone()))
+    }
+}
+
+impl<'a> std::convert::From<near_store::StoreOpener<'a>> for Store {
+    fn from(opener: near_store::StoreOpener<'a>) -> Self {
+        Self::from(&opener)
+    }
+}
+
+impl<'a> std::convert::From<&'a near_store::StoreOpener<'a>> for Store {
+    fn from(opener: &'a near_store::StoreOpener<'a>) -> Self {
+        Self(schema::Store::new(opener.open()))
+    }
+}
```

Comment:
These two looks superficial to me -- I'd rather the `.open()` was on the call-site. No super-strong arguments, but a bunch of weak ones:

* This dosen't fit semantics of the `From`, which is about converting between equivalent representations, rather than doing some action (opening)
* `From<&'_ T>` looks suspicious -- usually (but not always) `From` is from owned to owned. 
* `opener.open()` feels like something which should ideally return a `Result`, and, in that case, pushing open to the caller would be strictly beneficial.